pkg/envoy/proxy.go

```diff
@@ -245,6 +252,7 @@ func NewProxy(certCommonName certificate.CommonName, certSerialNumber certificat
 		xDSCertificateCommonName:   certCommonName,
 		xDSCertificateSerialNumber: certSerialNumber,
 		UUID:                       cnMeta.ProxyUUID,
+		Identity:                   cnMeta.ServiceIdentity,
```

Comment:
Could you also add an example of the Identity string here to improve readability. It would be convenient to know what this is expected to look like.

Reply:
I'm hesitant to add this, as I think Identity should be treated opaque. I'd even go as far to say that clients/callers shouldn't even know that it's just an aliased string, and treat it as an opaque object.

Reply:
Not documenting a sample value hinders readability for me. Even if it's opaque, a sample value in a comment is enough to clarify what can be expected here so it's easier to reason about it.

Reply:
ok, added